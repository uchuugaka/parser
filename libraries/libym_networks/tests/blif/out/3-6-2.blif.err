"3-6-2.blif" line: 4, column: 12 [ER_SYN16]: Syntax error in '.gate' statement.
Error in reading 3-6-2.blif
