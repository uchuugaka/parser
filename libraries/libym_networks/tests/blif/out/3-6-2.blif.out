.model test1
	["3-6-2.blif" line: 3, columns:1-6]
	["3-6-2.blif" line: 3, columns:8-12]
.gate inv4x begin
	["3-6-2.blif" line: 4, columns:1-5]
	["3-6-2.blif" line: 4, columns:7-11]
error_exit
